ledger-service/http-json-testing/src/main/scala/com/daml/http/HttpServiceTestFixture.scala

```diff
@@ -29,15 +30,15 @@ import com.daml.ledger.api.refinements.ApiTypes.ApplicationId
 import com.daml.ledger.api.refinements.{ApiTypes => lar}
 import com.daml.ledger.api.tls.TlsConfiguration
 import com.daml.ledger.api.v1.{value => v}
-import com.daml.ledger.client.{LedgerClient => DamlLedgerClient}
 import com.daml.ledger.client.configuration.{
   CommandClientConfiguration,
   LedgerClientConfiguration,
   LedgerIdRequirement,
 }
-import com.daml.ledger.participant.state.v1.SeedService.Seeding
+import com.daml.ledger.client.{LedgerClient => DamlLedgerClient}
```

Comment:
Please don't use import cleaning in any way. We don't have an idempotent approach.

Reply:
I know it's not always idempotent, but I did verify that reordering these imports won't affect any behavior.

There are large parts of the codebase where import ordering is standard (e.g. pretty much everything under _ledger_), so I'd rather not turn it off. If you like, I can revert this file.

Reply:
> I did verify that reordering these imports won't affect any behavior.

I don't think this is a good reason to add such hunks. I can do lots of import reordering that doesn't affect their behavior; even so, I'm still gumming up the review (and potentially forcing conflicts) with totally irrelevant diff lines.

> There are large parts of the codebase where import ordering is standard

I would say that it is more the case that they _look_ standard by the eye test, but do not actually match up to any specific standard. You will never reach an equilibrium with this approach, as things stand, but instead will add diff/conflict churn multiplied by however many people also do import reordering.

Reply:
That's fair. I agree that it's kind of a mess and not everyone follows the same conventions, uses the same version of the IDE, remembers to run the import reorganizer, or is in a situation where there is no equilibrium.

BRB gonna go write a tool to do this in a "standard" way.

Reply:
Alright, see you in a couple years 🤣